### DISCUSSION TITLE
DarkblowPB Website V2.0.0.e-08102022

### DISCUSSION THREAD

**Discussion by imamrasyid** [DarkblowPB/darkblowV42_Website#13] [General]
Stable Version

Changelog:
- Define Database Name In A New Library
- Define Access Level In A New Library
- Define New Library
- Define New Protection
- Delete Old Protection
- Delete Useless Code
- Improve Function
- Fixed Bugs
- Add License (BETA)

<hr /><em>This discussion was created from the release <a href='https://github.com/imamrasyid/darkblowV42_Website/releases/tag/latest-stable'>DarkblowPB Website V2.0.0.e-08102022</a>.</em>

**Comment by muhamadnaufan37** [comment]
can't register an account on the website, give definite instructions

**Reply by imamrasyid** [reply]
on it, this will be  fixed on next update
